lib/ogr/spatial_reference_mixins/importers.rb

```diff
@@ -5,6 +5,133 @@
 module OGR
   module SpatialReferenceMixins
     module Importers
+      def self.included(base)
+        base.extend ClassMethods
+      end
+
+      module ClassMethods
+        # @param code [Integer]
+        # @return [OGR::SpatialReference]
+        def new_from_epsg(code)
+          srs = new
+          srs.import_from_epsg(code)
+
+          srs
```

Comment:
Lately I've been preferring this pattern, what do you think?
```suggestion
          new.tap { |srs| srs.import_from_epsg(code) }
```

Reply:
Eh, it's obviously less lines, but kinda doesn't seem worth much to go change them all. ...and probably doesn't matter much (and maybe I'm wrong), but I imagine `tap` makes an extra allocation for the block. I lean towards leaving them unless you feel strongly about it.

Reply:
Nah, just mentioning it.